lib/plugins/inspec-compliance/lib/inspec-compliance/api.rb

```diff
@@ -293,7 +294,7 @@ def self.target_is_automate2_server?(url, insecure)
         if response.code == '400'
           Inspec::Log.debug(
             "Received 400 from #{url}#{automate_endpoint} - " \
-            'assuming target is a Chef Automate2 instance',
+            "assuming target is a #{Inspec::Dist::AUTOMATE_PRODUCT_NAME}2 instance",
```

Comment:
Should this be its own constant just in case we decide to rename something yet again? I don't know how far out that is...

Reply:
We just renamed everything, but the community distribution will have to choose something other than automate. At the very least we can remove the '2' here since the product is named 'Chef Automate'.

Reply:
Within this context are different methods for discovery of API endpoint type. Without to know all details, I saw it reasonable to leave the `2` as otherwise this debug message can't be differentiated from the message of `target_is_automate_server` below